UITests/AuthenticationManagerTests.swift

```diff
@@ -148,14 +122,11 @@ class AuthenticationManagerTests: KIFTestCase {
     }
 
     func testChangeRequirePasscodeInterval() {
-        setPasscode("1337", interval: .Immediately)
+        PasscodeUtils.setPasscode("1337", interval: .Immediately)
 
         openAuthenticationManager()
         tester().tapViewWithAccessibilityLabel("Require Passcode, Immediately")
 
-        tester().waitForViewWithAccessibilityLabel("Enter Passcode")
-        enterPasscodeWithDigits("1337")
-
```

Comment:
Quickest way round this for now is to change line 101 of AppMenuConfiguration inside `menuItemsForAppState` to
`case .HomePanels, .Loading:`